src/Team-Services-Bot.Api.UnitTests/Dialogs/ConnectDialogTests.cs

```diff
@@ -44,18 +42,22 @@ public ConnectDialogTests()
         [TestMethod]
         public async Task FirstTimeConnectionTest()
         {
-            var toBot = this.Fixture.CreateMessage();
-            toBot.From.Id = Guid.NewGuid().ToString();
-            toBot.Text = "connect anaccount";
+            var fromId = Guid.NewGuid().ToString();
+            var toBot1 = this.Fixture.CreateMessage();
+            toBot1.From.Id = fromId;
+            toBot1.Text = "Hi";
+
+            var toBot2 = this.Fixture.CreateMessage();
```

Comment:
Probably one of those "religious" debates, but I dislike var at the best of times. I will accept var where the type is explicit on the right hand side, but I really think it should not be used if the type is not clear by inspection. You can't necessarily program to an interface if you use var as well.